src/latexify/test_utils.py

```diff
@@ -109,19 +113,7 @@ def assert_ast_equal(observed: ast.AST, expected: ast.AST) -> None:
     Raises:
         AssertionError: observed and expected represent different ASTs.
     """
-    if sys.version_info.minor >= 9:
-        assert ast_equal(
-            observed, expected
-        ), f"""\
-AST does not match.
-observed={ast.dump(observed, indent=4)}
-expected={ast.dump(expected, indent=4)}
-"""
-    else:
-        assert ast_equal(
-            observed, expected
-        ), f"""\
-AST does not match.
-observed={ast.dump(observed)}
-expected={ast.dump(expected)}
-"""
+    indent = 4 if sys.version_info.minor >= 9 else None
+    assert ast_equal(
+        observed, expected
+    ), f"AST does not match. observed = {observed}\nexpected = {expected}"
```

Comment:
I looks this change is incomplete and should be reverted for now

Reply:
I removed the if condition because python always >=3.9 and revert the ast dump logic